tests/unit/modules/network/nxos/test_nxos_l3_interfaces.py

```diff
@@ -1044,3 +1044,26 @@ def test_replaced_tag(self):
         playbook["state"] = "replaced"
         set_module_args(playbook, ignore_provider_arg)
         self.execute_module(changed=True, commands=commands)
+
+    def test_ipv6_redirects(self):
```

Comment:
This test case seems to pass even without the changes in this PR. I don't think this covers the bug being solved.